### PR TITLE
Fix download structure

### PR DESCRIPTION
**Pull request by stringhandler**
Update downloads page to take into account the new S3 json format

### CODE REVIEW

**Review by delta1 — APPROVED** [review]
LGTM

File: assets/js/downloads.js [hunk 1]
```diff
@@ -36,79 +36,59 @@ function removeClass(elems) {
...
-    })
+    let linux = data.linux;
+    let windows = data.windows;
+    let osx = data.osx;
+    let libWallet = data.libWallet;
+    let sortedFiles = { linux, windows, osx, libWallet };
+
+    Object.keys(sortedFiles).forEach((os) => {
+      renderBinaries(sortedFiles[os], os);
+    });
```
Comment (Krakaw) [inline assets/js/downloads.js:62]:
```suggestion
    Object.keys(data).forEach((os) => {
      renderBinaries(data[os], os);
    });
```

Comment (stringhandler) [reply assets/js/downloads.js:62]: I'm not sure if this is correct. It will include `latest` then..

Comment (Krakaw) [reply assets/js/downloads.js:62]: Yeah, my bad, it was mostly to fix `libWallet` it's all lowercase.

**Pull request merged**